projects/plugins/jetpack/modules/likes.php

```diff
@@ -367,10 +381,10 @@ function load_admin_css() {
 	}
 
 	/**
-	* Load the JS required for loading the like counts.
-	*/
-	function enqueue_admin_scripts() {
-		if ( empty( $_GET['post_type'] ) || 'post' == $_GET['post_type'] || 'page' == $_GET['post_type'] ) {
```

Comment:
This is the last thing that needs addressing, but it doesn't look like a nonce is set and I'm honestly not sure where in this case we'd set one.